kernel/univ.mli

```diff
@@ -268,6 +268,8 @@ sig
   type t
 
   val make : Names.Name.t array -> Instance.t constrained -> t
+  (** The name array must be the same length as the instance, and the
+      instance must be all global names (no Set, no var) *)
```

Comment:
This comment must be tweaked since we currently accept vars.

Reply:
changed